spark/v3.5/spark/src/test/java/org/apache/iceberg/spark/actions/TestRewritePositionDeleteFilesAction.java

```diff
@@ -275,7 +275,7 @@ public void testRewriteFilter() throws Exception {
             .execute();
 
     List<DeleteFile> newDeleteFiles = except(deleteFiles(table), deleteFiles);
-    assertThat(newDeleteFiles).as("Should have 4 delete files").hasSize(2);
+    assertThat(newDeleteFiles).as("Should have 2 delete files").hasSize(2);
```

Comment:
Question, should we just remove the number here, how about, 'delete files'?

Wont the assert message in the end already have the expected value from hasSize()?

Reply:
@ebyhr @huaxingao @szehon-ho  I think this suggestion is reasonable, but there are many similar assertions in the existing uts. Should we work together to update them as well?

Reply:
Done. The updated assert message is as follows:

[Delete files] 
Expected size: 1 but was: 2 in: